logspace-ai/langflow
# Dark Mode and API Modal

Changes Made
Updated tailwind dark mode classes for previously unimplemented objects
Added a new modal to explain how to use the API of the flow
Why These Changes Were Made
These changes are designed to improve the user experience of our project and make it easier for developers to work with our API. We believe that these updates will be well-received by our users and will help us to continue to build a strong and engaged community around our project.

How to Test
To test these changes, simply clone this branch and run the application. You should notice the updated tailwind dark mode classes and the new modal that explains how to use the API of the flow.

Thank you for taking the time to review our pull request. We look forward to your feedback and hope to see these changes merged into the main branch soon.
